ui/src/createExperimentationService.ts

```diff
@@ -45,20 +45,26 @@ export async function createExperimentationService(ctx: vscode.ExtensionContext,
 class ExperimentationServiceAdapter implements IExperimentationServiceAdapter {
     public wrappedExperimentationService?: tas.IExperimentationService;
 
+    /**
+     * @deprecated Use `getCachedTreatmentVariable<boolean>('flight-name') instead

```

Comment:
We don't necessarily have to deprecate these, but the `vscode-tas-client` library did deprecate their versions of these methods. I thought it best to match but I don't feel strongly, so please chime in if you feel differently!